src/views/statutory/EditCandidate.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <div class="tile is-ancestor ">
     <div class="tile is-child">
+      <span class="notification is-info" v-if="$route.params.candidate_id">
+        To withdraw your candidature, you should send an email to the <a href="mailto:jc@aegee.eu">Juridical Commission</a>.
```

Comment:
```suggestion
        To withdraw your candidature, you should send an email to the <a href="mailto:juridical@aegee.eu">Juridical Commission</a>.
```
I think both work, but juridical is also the one we use in statutory